packages/client/src/comp/states/Georgia.tsx

```diff
@@ -8,26 +8,37 @@ import { SignatureBase, StatelessInfo, NoSignature } from './Base'
 import Select from 'muicss/lib/react/select'
 import Option from 'muicss/lib/react/option'
 
+const georgiaPrimary = process.env.REACT_APP_GEORGIA_PRIMARY
+
 export const Georgia = () => {
   const partyRef = useControlRef<Input>()
 
   const enrichValues = (baseInfo: StatelessInfo): NoSignature<GeorgiaInfo> | null => {
-    const party = partyRef.value()
-    if (!isGeorgiaParty(party)) return null
+    if (georgiaPrimary) {
```

Comment:
I just think it would make more sense to have 'if (false) {' instead of an environment variable.  We won't be going back to this for a long time.